### PR TITLE
Add verifybamid2

### PR DESCRIPTION
**Pull request by A97paupic**
# Description and reviewer info

Add a verifybamid2 process with a if clause to separate between wgs and panel data input. In addition the data used by the verifybamid2 software and container has been added to the config.  

## Type of change

- [ ] Documentation
- [ ] Patch
- [x] Minor change
- [ ] Major change 

# Checklist

- [x] Self-review of my code
- [x] Update the CHANGELOG
- [ ] Tag the latest commit (vX.Y.Z format)
- [ ] Log samples used for testing in the `Verification_samples_log` Excel sheet

## Documentation
- [ ] At least one other person has reviewed my changes (not required for trivial changes)

## Patch
- [ ] Stub run completes without errors or new warnings
- [ ] At least one other person has reviewed and approved my code (not required for trivial changes)

## Major / Minor change
- [x] Stub run completes without errors or new warnings
- [ ] `onco` run finishes without any new warnings/errors and the results can 
       be loaded into scout
- [x] `wgs` single run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [ ] `wgs` trio run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [x] At least one other person has reviewed and approved my code
- [x] I have made corresponding changes to the documentation (software versions, etc.)

# Test/review documentation

## Review performed by

- [ ] Alexander
- [x] Jakob
- [ ] Paul
- [ ] Ryan
- [ ] Viktor

(Add if missing)

## Testing performed by

- [ ] Alexander
- [ ] Jakob
- [x] Paul
- [ ] Ryan
- [ ] Viktor

### CODE REVIEW

File: CHANGELOG.md
Comment (Jakob37) [inline CHANGELOG.md:3]: This is probably more of a minor change than a patch. We are adding new functionality here.

Comment (Jakob37) [reply CHANGELOG.md:3]: I.e. `3.11.1`

File: configs/nextflow.hopper.config
Comment (Jakob37) [inline configs/nextflow.hopper.config:151]: Please update the variable name to make it clear what process it is intended for

I.e. `params.verifybamid2_svdprefix`

Otherwise it will be confusing when reading the config to know what this variable is intended for.

File: main.nf
Comment (Jakob37) [inline main.nf:653]: Are these intended to be published? Or not published, just part of the QC outputs. If intended to be published in the long run, I would uncomment before merge. If unsure, we could discuss in the RD BNF meeting tomorrow.

File: main.nf
Comment (Jakob37) [inline main.nf:667]: This code could be made nicer by reusing the common part, and just having the `DisableSanityCheck` be inserted when running in non WGS mode.

But anyways, not a big deal, and now it is tested as is. So not a request for change, just a thought for future processes.

File: main.nf
Comment (Jakob37) [inline main.nf:706]: Have you verified that this looks correct in the `version.yaml` output?

File: main.nf
Comment (Jakob37) [inline main.nf:671]: What about `${genome_file}`, to keep things consistent

**Review by Jakob37 — CHANGES_REQUESTED** [review]
Great job 💪 Just some minor points and questions.

Don't forget to add the version to the version document.

I would like to take a final quick look before merge when updated.

File: main.nf
Comment (Jakob37) [reply main.nf:706]: Here is the document with all versions. This version should be included here as well:

https://github.com/Clinical-Genomics-Lund/nextflow_wgs/blob/master/docs/list_of_all_used_software.md

File: CHANGELOG.md
Comment (A97paupic) [reply CHANGELOG.md]: Done!

File: main.nf
Comment (A97paupic) [reply main.nf:653]: Dont know really, depends what we would like to do.

File: main.nf
Comment (A97paupic) [reply main.nf:667]: I was just looking on how similar processes were built and took inspiration but if you think its redundant the way it is written I can change it?

File: main.nf
Comment (A97paupic) [reply main.nf:706]: Yes looks good. I will add it to the list of versions asap,

File: main.nf
Comment (A97paupic) [reply main.nf]: Done!

File: main.nf
Comment (Jakob37) [reply main.nf:653]: Yes, maybe we can discuss this tomorrow before merge

File: main.nf
Comment (Jakob37) [reply main.nf:667]: I think it is fine to leave as is here!

File: main.nf
Comment (Jakob37) [reply main.nf:706]: (Unresolved, so you can resolve when actually resolved ...)

**Comment by Jakob37** [comment]
Looks nice! But please add the version to the version document before merge (that is part of the same repo, so it is just to change that `.md` file). I'll approve when I see that it is in.

File: main.nf
Comment (Jakob37) [reply main.nf:653]: (Or after merge, this could come later)

**Comment by A97paupic** [comment]
The version document is now updated with verifybamid2!

**Comment by Jakob37** [comment]
> The version document is now updated with verifybamid2!

Great!

I don't see it in the PR here though. Have you forgotten to push the changes?

**Pull request merged**
